src/ucp/rndv/proto_rndv.c

```diff
@@ -630,6 +644,8 @@ ucp_proto_rndv_bulk_init(const ucp_proto_multi_init_params_t *init_params,
     const char *proto_name;
     ucs_status_t status;
 
+    ucp_proto_rndv_report_fast_cmpl(init_params);
```

Comment:
seems we remove FAST_CMPL when we handle RTR. So in which case we sould still have FAST_CMPL here?
maybe it should be an assertion?

Reply:
AFAIU that can happen if customer set FAST_CMPL flag for receive operation. In that case RNDV-get would be initialized with FAST_CMPL flag

Reply:
so maybe make the warning more clear, or move it to tag_recv_nbx, or mask-out FAST_CMPL flag during tag_recv?

Reply:
I can move this warning to `tag_recv_nbx` (or maybe `ucp_proto_rndv_receive_start`) but then it would be extra operation on the fast path. And if we want to print a waring/debug message in that case it would cost extra `if` on fast path. Are we OK with that?

Reply:
ok, let's keep here